templates/rhsm-katello-reconfigure.erb

```diff
@@ -83,6 +83,13 @@ else
       --server.port="$PORT" \
       --rhsm.repo_ca_cert="%(ca_cert_dir)s$KATELLO_SERVER_CA_CERT" \
       --rhsm.baseurl="$BASEURL"
+
+    # Older version of subscription manager might not recogniza
+    # report_package_profile and package_profile_on_trans options
+    # set them separately and redirect out & error to /dev/null
+    # to fail silently
+    subscription-manager config --rhsm.package_profile_on_trans=1 &> /dev/null
```

Comment:
you may want to do 2&>1 to send all output (both stderr and stdout) to /dev/null